packages/lesswrong/server/recombee/client.ts

```diff
@@ -612,7 +612,8 @@ const recombeeApi = {
     const reqIsLoadMore = helpers.isLoadMoreOperation(lwAlgoSettings);
     const includedCuratedPostIds = curatedPostIds.filter(id => !curatedPostReadStatuses.find(readStatus => readStatus.postId === id));
     const includedStickiedPostIds = stickiedPostIds.filter(id => !manuallyStickiedPostReadStatuses.find(readStatus => readStatus.postId === id))
-    const excludeFromLatestPostIds = [...includedAboutPagePostId, ...includedCuratedPostIds, ...includedStickiedPostIds];
+    const manualExcludedPostIds = lwAlgoSettings.excludedPostIds ?? [];
+    const excludeFromLatestPostIds = [...includedAboutPagePostId, ...includedCuratedPostIds, ...includedStickiedPostIds, ...manualExcludedPostIds];
```

Comment:
This might break other stuff, since `includedTopOfListPostIds` depends on the length of `excludeFromLatestPostIds`.  (Probably `includedTopOfListPostIds` should be the first three values here, and then `excludeFromLatestPostIds` should actually derive from that + the manually excluded post ids...?)